modules/serverless-known-limitations-serving.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+//
+// * /knative-serving/scalability-and-performance-serving.adoc
+
+
+:_mod-docs-content-type: CONCEPT
+[id="serverless-known-limitations-serving_{context}"]
+= Known limitations of OpenShift Serverless Serving
+
+The maximum number of Knative Services that can be created is 3,000. This corresponds to the OpenShift Container Platform Kubernetes services limit of 10,000, since 1 Knative Service creates 3 Kubernetes services.
```

Comment:
```suggestion
The maximum number of Knative Services that can be created is 3,000. This corresponds to the {product-title} Kubernetes services limit of 10,000, since 1 Knative Service creates 3 Kubernetes services.
```